### PR TITLE
Cmungall main

### PR DESCRIPTION
**Pull request by ramonawalls**
merging with the source

### CODE REVIEW

**Comment by ramonawalls** [comment]
This PR includes changes I made to match the new format of MIxS6. @cmungall if you merge, we can make the version in the GSC org the source.

**Comment by ramonawalls** [comment]
It also includes changes to the python script, which are just commenting out sections for columns we no longer include. I did not delete any code.

Also includes the TSV files that @only1chunts added to the repo from his clean spreadsheet.

File: downloads/mixs6.tsv [hunk 1]
```diff
@@ -0,0 +1,2 @@
+<!DOCTYPE html><html lang="en"><head><meta name="description" content="Web word processing, presentations and spreadsheets"><meta name="viewport" content="width=device-width, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0, user-scalable=0"><link rel="shortcut icon" href="//docs.google.com/favicon.ico"><title>Page Not Found</title><meta name="referrer" content="origin"><link href="//fonts.googleapis.com/css?family=Product+Sans" rel="stylesheet" type="text/css" nonce="OkwGdLmsoRzP3RI3CdvFZA"><style nonce="OkwGdLmsoRzP3RI3CdvFZA">/* Copyright 2021 Google Inc. All Rights Reserved. */
```
Comment (cmungall) [inline downloads/mixs6.tsv:1]: This doesn't look right...

File: Makefile [hunk 2]
```diff
@@ -223,23 +232,16 @@ target/rdf/%.model.ttl: $(SCHEMA_DIR)/%.yaml $(PKG_DIR)/jsonld/%.model.context.j
...
-
 
 # ---------------------------------------
 # TSVs from google drive
 # ---------------------------------------
 # for seeding
 
 downloads/mixs6.tsv:
-	curl -L -s 'https://docs.google.com/spreadsheets/d/1QDeeUcDqXes69Y2RjU2aWgOpCVWo5OVsBX9MKmMqi_o/export?format=tsv&gid=345753674' > $@
+	curl -L -s 'https://docs.google.com/spreadsheets/d/165AHU4Px4S1fFFIqsvffWFlD1RUFmH609QbuDl4pASk/export?format=tsv&gid=345753674' > $@
```
Comment (cmungall) [inline Makefile:242]: decided on call - we will go back to previous

**Pull request merged**